Sandbox/Sandbox.cpp

```diff
@@ -3,52 +3,78 @@
 #include <glm/gtc/matrix_transform.hpp>
 
 SandboxLayer::SandboxLayer()
-	: Layer("Sandbox")
+    : Layer("Sandbox")
 {}
 
 void SandboxLayer::onCreation()
 {
-	m_vertexBuffer = std::make_shared<Aquarius::VertexBuffer>(
-		m_triangleVertices.data(), m_triangleVertices.size() * sizeof(float));
-
-	m_indexBuffer = std::make_shared<Aquarius::IndexBuffer>(m_indices.data(), m_indices.size());
-
-	m_bufferLayout = std::make_shared<Aquarius::BufferLayout>(std::initializer_list<Aquarius::VertexElement>{
-		Aquarius::VertexElement(Aquarius::ShaderType::Float, 2, false),
-		Aquarius::VertexElement(Aquarius::ShaderType::Float, 3, false),
-		Aquarius::VertexElement(Aquarius::ShaderType::Float, 2, false)
-	});
-
-	m_vertexArray = std::make_shared<Aquarius::VertexArray>(m_vertexBuffer, m_indexBuffer, *m_bufferLayout);
-	m_vertexArray->activate();
-	Aquarius::Renderer::Init();
-	m_texture = std::make_shared<Aquarius::Texture>("Assets/container.jpg");
-	m_texture->bind(0);
+    m_ShaderProgram = std::make_shared<Aquarius::Shader>("Sandbox/Shaders/modifiableVertexShader.glsl", 
+                                                         "Sandbox/Shaders/modifiableFragmentShader.glsl");
```

Comment:
Ok nice so these shaders are for playing around with and now the quad renderer shader is in the engine. Sweeet

Reply:
Yeah we can do this temporarily, not sure how to handle shaders tbh hahaha